### DISCUSSION TITLE
Concerns with Supabase Edge Functions Deno support for third-party SDKs

### DISCUSSION THREAD

**Discussion by KhaledGabr** [supabase/supabase#6272] [General]
I am currently a bit uncertain on whether I should be using Supabase functions since most client SDKs are supporting NodeJS only and not Deno runtime. (Ex: payments, email, etc). 

It's a bit sad since I would want to use supabase for everything especially since the function as a service approach works well for me, but in the same time I would want to make use of libraries that will speed my development instead of crafting http requests from scratch. 

L

**Comment by thorwebdev** [comment]
Hey @KhaledGabr, any specific examples where you're missing Deno support? E.g. see here for an example making payments with Stripe: https://github.com/supabase-community/expo-stripe-payments-with-supabase-functions
